### ISSUE TITLE
Loss is NaN in training

### ISSUE THREAD

**Issue by mappro6** [HLinChen/VCR-GauS#7]
@HLinChen Hi, I used the dataset and corresponding script in the readme.md for training, but when training the Barn scene in the TNT dataset, the Loss was NaN, which caused the training to stop. Have you encountered such a situation?
![image](https://github.com/user-attachments/assets/a3cb400b-f4a4-4ac6-8492-65e317d7b2b3)

**Comment by YangDeng007** [comment]
I met the same problem due to the wrong loss. But I haven't solved it.
![image](https://github.com/user-attachments/assets/adfc26a4-a039-4a16-8e9b-7c2c0d3ed301)

**Comment by HLinChen** [comment]
I didn’t meet this question. Can you check in which line in our code the problem meets?

**Comment by mappro6** [comment]
I haven’t figured out why yet, at some iteration the loss becomes so large that it becomes NaN, and this process seems random, so I can’t debug it.

**Comment by YangDeng007** [comment]
> I haven’t figured out why yet, at some iteration the loss becomes so large that it becomes NaN, and this process seems random, so I can’t debug it.

Yes, I met the same problem, the loss becomes so large like this
![image](https://github.com/user-attachments/assets/159d52c1-9345-4834-ae46-66b0c9bcf2eb)

**Comment by HLinChen** [comment]
The problem may be the environment. I uploaded a built anaconda environment [here](https://huggingface.co/hanlin-chen/VCR-GauS/resolve/main/vcr.zip?download=true); you can download it and unzip and put it in your_anaconda_path/envs/ .
